bin/v-backup-user

```diff
@@ -83,7 +83,7 @@ get_user_disk_usage() {
 			| awk -F "DOMAIN='" '{print $2}' | cut -f 1 -d \')
 
 		for domain in $domains; do
-			check_exl=$(echo "$mail_exclusions" | tr ',' '\n' | grep "^$domain$")
+			check_exl=$(echo "$mail_exclusions" | tr ',' '\n' | grep "^$domain\|\*$")
```

Comment:
@youradds  for mail domains this doesn't work:

If I use:  domain:info I get:

-- MAIL --
2023-01-28 11:11:21 domain.com
2023-01-28 11:11:21 excluding mail account info
2023-01-28 11:11:21 include email accounts: 
2023-01-28 11:11:21 *** 1 domain ***

If I use: *:info

-- MAIL --
2023-01-28 11:10:47 excluding dev.eris.nu
2023-01-28 11:10:47 *** 0 domains ***

Reply:
Thanks. What should the format for an exclusion be?  I'm doing a test with *:info, but it gives: (with some tweaks and debug)

```
-- MAIL --
TEST: *:info
NOW: ultranerds.co.uk:info
2023-01-28 11:10:04 excluding ultranerds.co.uk
2023-01-28 11:10:04 *** 0 domains ***
```

I would have expected it to backup ultranerds.co.uk, buy ignore info@

Reply:
Yes the same but it did ignore everything.. 

If you would use:
ultranerds.co.uk:info it does backup the mail domain but ignores info@ ultranerds.co.uk

Reply:
Hmm I tested it with just:

`ultranerds.co.uk:info`

With debug $check_exl (line 479), is:

`NOW: ultranerds.co.uk:info`

And then:

```
   if [ -z "$check_exl" ]; then
			mail_list="$mail_list $domain"
		else
			echo "$(date "+%F %T") excluding $domain" | tee -a $BACKUP/$user.log
		fi
```

If I add some debug in, the if [] doesn't seem to trigger:

```
if [ -z "$check_exl" ]; then
			echo "HERE"
			mail_list="$mail_list $domain"
		else
			echo "$(date "+%F %T") excluding $domain" | tee -a $BACKUP/$user.log
		fi
```

I just get:

2023-01-28 11:21:02 excluding ultranerds.co.uk

I'm not sure why though? (as $check_exl says it has "ultranerds.co.uk:info" as the value)